package.json

```diff
@@ -67,7 +67,7 @@
     "scrollspy"
   ],
   "dependencies": {
-    "consolidated-events": "^1.1.0",
+    "consolidated-events": "^1.1.0 || ^2.0.0",
```

Comment:
Why didn’t you just upgrade to 2.0.0?

Reply:
I want to make it easier for people who have to deal with breaking changes in consolidated-events to upgrade more gradually. Also, there's no reason in this package for us to not support v1, so might as well.